proxy/http/HttpTransact.h

```diff
@@ -1121,7 +1121,7 @@ class HttpTransact
       via_string[MAX_VIA_INDICES]              = '\0';
 
       memset(user_args, 0, sizeof(user_args));
-      memset(&host_db_info, 0, sizeof(host_db_info));
+      memset((void *)&host_db_info, 0, sizeof(host_db_info));
```

Comment:
If we want to consolidate to ink_zero I think we should put that in a separate ticket-- I can't replace every instance I run into in this PR-- the change set is already getting pretty big ;)